timesync/converter.go

```diff
@@ -24,3 +24,7 @@ func (lc LayerConv) TimeToLayer(t time.Time) types.LayerID {
 func (lc LayerConv) LayerToTime(id types.LayerID) time.Time {
 	return lc.genesis.Add(time.Duration(id.Uint32()) * lc.duration)
 }
+
+func (lc LayerConv) DurationToLayers(duration time.Duration) uint32 {
+	return uint32(duration.Nanoseconds() / lc.duration.Nanoseconds())
```

Comment:
how is this supposed to be used?
say layer time is 10 sec. duration is 234. number of layers is 23.
how does the client use this API?

Reply:
It transforms time duration to the number of layers it spans. It is described in the interface definition:
>// Convert time duration to number of layers it spreads

Correct, if the layer duration is 10s and the duration passed in is 234s, then it outputs 23.

Reply:
my main thought this, as an API is not generic enough and easy to misuse.

say numLayers := lc.DurationToLayers(234s)
if the client want to wait for `duration` to elapse, then waiting till currentLayer+numLayers != 234s.

i think this should just a line of code at the call site. `duration/types.LayerDuration()` and the context in the call site 
we don't currently have `types.LayerDuration()` but you can follow the setup for `types.GetLayersPerEpoch()` from app.Config value.

Reply:
Removed